igvc_platform/src/motor_controller/motor_controller.cpp

```diff
@@ -277,6 +279,14 @@ void MotorController::publishResponse(const ResponseMessage& response)
   enc_msg.duration = response.dt_sec;
   enc_msg.header.stamp = ros::Time::now() - ros::Duration(response.dt_sec);
   enc_pub_.publish(enc_msg);
+
+  // Publish motor output
+  igvc_msgs::motor_control motor_control;
+  motor_control.left = static_cast<uint8_t>(response.left_output);
+  motor_control.right = static_cast<uint8_t>(response.right_output);
+  motor_control.duration = ros::Duration(response.dt_sec);
+  motor_control.header.stamp = ros::Time::now() - motor_control.duration;
```

Comment:
I'm not sure why you subtracted duration from the current time. What do you want the header time to represent?

Reply:
You're right this should just be `ros::Time::now()`.